src_c/math.c

```diff
@@ -4413,6 +4413,37 @@ com_descr_get(PyObject *self, PyObject *obj, PyObject *type)
     return PyMethod_New(com->obj_callable, obj);
 }
 
+/* python > 3.12 */
+#if PY_VERSION_HEX > 0x030C0000
+#ifndef _PyArg_NoKeywords
```

Comment:
I guess this `#ifndef` is no longer necessary. The idea was that it would prevent the code from existing on Python < 3.13, but apparently, it did not work properly everywhere.

Reply:
Yeah. It can't hurt I don't think.